conf/machine/raspberrypi4-64.conf

```diff
@@ -34,3 +34,6 @@ KERNEL_BOOTCMD ?= "booti"
 RPI_EXTRA_CONFIG ?= "\n# Force arm in 64bit mode. See: https://github.com/raspberrypi/firmware/issues/1193.\narm_64bit=1"
 
 ARMSTUB ?= "armstub8-gic.bin"
+
+# disable vc4 for graphics to fall back using userland (dagerous!)
+DISABLE_VC4GRAPHICS = "1"
```

Comment:
@leckijakub Why **exactly** is it necessary? Do we have some error messages if it was not set?